packages/table-react/src/TableRow.tsx

```diff
@@ -28,13 +29,19 @@ export function TableRow(props: Props) {
     let rowModifierClasses = "";
 
     if (isAnchorRowData(row)) {
-        onClick = () => {
-            if (row.onRowClick) {
-                row.onRowClick(row.href);
-            } else {
-                window.location.href = row.href;
-            }
-        };
+        if (row.customRowClick !== undefined) {
+            onClick = () => {
+                row.customRowClick && row.customRowClick();
```

Comment:
This is exactly the same logic as `row.onRowClick`. Why can't `onRowClick` be used?